### PR TITLE
Add controller/services/dto/tests/view files for Meal and MealGroup

### PR DESCRIPTION
**Pull request by antooa**

### CODE REVIEW

File: SushiRunner.Data/Repositories/OrderRepository.cs [hunk 1]
```diff
@@ -18,7 +18,8 @@ public OrderRepository(ApplicationDbContext context)
 
         public IEnumerable<Order> GetList()
         {
-            return _context.Orders;
+            return _context.Orders.Include(entity => entity.Items).ThenInclude(item => item.Meal)
```
Comment (ojles) [inline SushiRunner.Data/Repositories/OrderRepository.cs]: Start each function from new line like this:

```
_context_Orders
    .Include(...)
    .ThenInclude(...)
    .ToList()
    .DoSomethingElse();
```

File: SushiRunner.Data/Repositories/OrderRepository.cs [hunk 2]
```diff
@@ -32,7 +33,7 @@ public Order Get(long id)
         {
             return _context.Orders
                 .AsNoTracking()
-                .Include(entity => entity.Items)
+                .Include(entity => entity.Items).ThenInclude(item => item.Meal)
```
Comment (ojles) [inline SushiRunner.Data/Repositories/OrderRepository.cs]: Start function `ThenInclude` from new line

File: SushiRunner.Services/Dto/MealDTO.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+{
+    public class MealDTO
+    {
+        public long Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public double Weight { get; set; }
+        public string ImagePath { get; set; }
+        public int Price { get; set; }
+        public long MealGroupId { get; set; }
```
Comment (ojles) [inline SushiRunner.Services/Dto/MealDTO.cs]: No need for `MealGroupId` here. It's just used to map relationships in entities

File: SushiRunner.Services/MealService.cs [hunk 4]
```diff
@@ -45,6 +53,20 @@ public void Delete(long id)
             _repository.Save();
         }
 
+        public IEnumerable<MealDTO> GetByGroup(string groupName)
+        {
+            var meals = _repository.Search(m => m.MealGroup.Name.Equals(groupName));
+            
+            return meals.Select(meal => _mapper.Map<Meal, MealDTO>(meal)).ToList();
```
Comment (ojles) [inline SushiRunner.Services/MealService.cs:59]: Maybe just:

```
return _repository.Search(...)
    .Select(meal => _mapper.Map<Meal, MealDTO>(meal))
    .ToList();
```

File: SushiRunner.Services/MealService.cs [hunk 5]
```diff
@@ -45,6 +53,20 @@ public void Delete(long id)
...
+        }
+
+        public IEnumerable<MealDTO> GetByGroupId(long id)
+        {
+            var meals = _repository.Search(m => m.MealGroup.Id.Equals(id));
+
+            return meals.Select(meal => _mapper.Map<Meal, MealDTO>(meal)).ToList();
```
Comment (ojles) [inline SushiRunner.Services/MealService.cs:65]: Same as previous commend

**Review by ojles — CHANGES_REQUESTED** [review]
Looks good to me. I'll finish my review tomorrow and merge this request

File: SushiRunner.Services/MealService.cs (hunk 4 above)
Comment (antooa) [reply SushiRunner.Services/MealService.cs:59]: Oles! Do you remember any Svyatoslav Volodymyrovych lessons? One action - one row.

**Pull request merged**